cache/lru_test.go

```diff
@@ -56,6 +56,11 @@ func TestLRUCache_StoreFetch(t *testing.T) {
 		# TYPE cache_memory_requests_total counter
 		cache_memory_requests_total{name="test"} 4
 	`)))
+
+	err = lru.Delete(context.Background(), "buzz")
```

Comment:
[nit] Please rename the test to `TestLRUCache_StoreFetchDelete`. Please also add an assertion on metrics at the end of this test. It should be something like this:

```
	require.NoError(t, testutil.GatherAndCompare(reg, strings.NewReader(`
		# HELP cache_memory_items_count Total number of items currently in the in-memory cache.
		# TYPE cache_memory_items_count gauge
		cache_memory_items_count{name="test"} 2
		# HELP cache_memory_hits_total Total number of requests to the in-memory cache that were a hit.
		# TYPE cache_memory_hits_total counter
		cache_memory_hits_total{name="test"} 2
		# HELP cache_memory_requests_total Total number of requests to the in-memory cache.
		# TYPE cache_memory_requests_total counter
		cache_memory_requests_total{name="test"} 4
	`)))
```